aiden-web/Dockerfile

```diff
@@ -26,6 +26,5 @@ COPY aiden_project ./
 COPY manage.py ./
 
 EXPOSE 8000
-CMD poetry run python manage.py makemigrations && \
-    poetry run python manage.py migrate && \
+CMD poetry run python manage.py migrate && \
```

Comment:
this did not work for me, I had to put back the makemigrations

Reply:
me too

Reply:
I reverted, had the same problem